lib/Frontend/CompilerInvocation.cpp

```diff
@@ -675,7 +665,7 @@ bool FrontendArgsToOptionsConverter::computeFallbackModuleName() {
       !llvm::sys::fs::is_directory(outputFilenames[0]);
   std::string nameToStem = isOutputAUniqueOrdinaryFile
                                ? outputFilenames[0]
-                               : Opts.Inputs.getFilenameOfFirstInput();
+                               : Opts.Inputs.getFilenameOfFirstInput().str();
```

Comment:
IIRC there's an implicit conversion here too, so this change isn't strictly necessary.

Reply:
Removing the .str() yields an error:
conditional expression is ambiguous; 'const std::__1::basic_string<char>' can be converted to 'llvm::StringRef' and vice versa

Reply:
Oh, hm, right, okay.